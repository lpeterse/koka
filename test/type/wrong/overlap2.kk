// test overlap error
fun g( x : int ) { x }
fun g(x) { x }
