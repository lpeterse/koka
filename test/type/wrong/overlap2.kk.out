test/type/wrong/overlap2.kk(3, 5): error: definition g overlaps with an earlier definition of the same name
 type : forall<a> (x : a) -> a
 overlaps: (x : int) -> int
 because : definitions with the same name must differ on the argument types